src/main/java/org/folio/circulation/domain/notice/schedule/DueDateScheduledNoticeHandler.java

```diff
@@ -149,8 +154,14 @@ public static DueDateScheduledNoticeHandler using(Clients clients, DateTime syst
 
     JsonObject loanNoticeContext = TemplateContextUtil.createLoanNoticeContext(loan);
 
-    return patronNoticeService.acceptScheduledNoticeEvent(
-      notice.getConfiguration(), relatedRecords.getUserId(), loanNoticeContext)
+    NoticeLogContext noticeLogContext = NoticeLogContext.from(loan)
+      .withTemplateId(notice.getConfiguration().getTemplateId())
+      .withTriggeringEvent(notice.getTriggeringEvent().getRepresentation());
+
+    return noticePolicyRepository.lookupPolicyId(loan.getItem(), loan.getUser())
```

Comment:
Why is it necessary to lookup the notice policy here, why is that not done when the notice is being sent (which should be separate to this change)?